src/ext/java/org/opentripplanner/ext/bikerentalservicedirectory/BikeRentalServiceDirectoryFetcherParameters.java

```diff
@@ -0,0 +1,35 @@
+package org.opentripplanner.ext.bikerentalservicedirectory;
+
+import java.net.URI;
+
+public class BikeRentalServiceDirectoryFetcherParameters {
```

Comment:
This class need java doc. What is is used for, what are the fields(JavaDoc on getters). What is legal values, defaults and required/optional.